game-scripts/dontfall.lua

```diff
@@ -0,0 +1,155 @@
+--
+--
+-- Don't Fall
+--
+--
+
+-- First thing to do in a game script: import some items!
+-- (including the map)
+-- Complete item names should be used: <repository>.<item>
+-- Official items can be imported with just <item> (shortcut for official.<item>)
+-- Imported items become available under Resources (or R shortcut):
+-- Resources.<repository>.<item> (or R.<repository>.<item>)
+Import (
+    "aduermael.dont_fall_2" -- item used as map
+)
+
+-- Events are objects that can be sent to other players, 
+-- and/or to the Server.
+-- There are pre-defined event types, documented here:
+-- https://docs.particubes.com/reference/EventType
+-- We can declare custom ones this way:
+EventType.playerDied = EventType:New() 
+
+-- Set the map
+-- (R is a shortcut for Shared.Resources)
+Map.Set(R.aduermael.dont_fall_2)
+
+-- Config is also exposed by the engine, it contains 
+-- a few pre-configured values. (like Config.DefaultJumpStrength)
+-- We can also use it to store our own things:
+Config.fireRate = 0.1
+
+Local.Player.triggerOn = false
+Local.Player.firedOnce = false
+Local.Player.fireDT = 0.0
+
+-- Local.Player represents the local player.
+-- Local.Player.Jump is nil (non existent) by default but we
+-- can assign a function, defining how the players jumps
+-- in the game. 
+-- NOTE: players can jump differently based on the item
+-- they're holding for example.
+Local.Player.Jump = function (player)
+    -- Test if player is on ground before changing velocity,
+    -- otherwise, player could jump while in the air. :D
+    if player.IsOnGround then
+      player.Velocity.Y = Config.DefaultJumpStrength
+    end
+end
```

Comment:
We should not be able to jump in `Don't fall`

Reply:
Thank you ! 😲